### ISSUE TITLE
Doesn't seem to be working on Ubuntu 16.04

### ISSUE THREAD

**Issue by x-ji** [miracle2k/sendtokindle#2]
After I tried to open a mobi file with sendtokindle, its application icon flashes for a while in the icon bar, but no window is displayed and the icon soon disappears as well. It doesn't seem to do anything.

**Comment by joseboretto** [comment]
Me too. Do you know any way to fix it?

**Comment by silentjay** [comment]
Looks like this app is dead and abandoned

**Comment by x-ji** [comment]
Currently I just send via email anyways.

**Comment by silentjay** [comment]
@x-ji  yeah that's what I ended up doing. I did get sendtokindle vaguely working but it was very buggy and gmail wasn't happy about it

**Comment by stuaxo** [comment]
Hi,
   Try the my WIP version in the PR here:

https://github.com/miracle2k/sendtokindle/pull/3

It's a bit late at night and I haven't worked out which details are needed to get this setup, I can say the GUI comes up though, on Ubuntu 20.04

![Screenshot from 2020-07-16 01-21-31](https://user-images.githubusercontent.com/179677/87612568-d1168f00-c702-11ea-967c-5a89bf71bcda.png)
